lib/helpbase.gi

```diff
@@ -498,7 +498,7 @@ InstallGlobalFunction(HELP_ADD_BOOK, function( short, long, dir )
   # (looks a bit lengthy)
   sortfun := function(a, b)
     local main, pa, pb;
-    main := ["tutorial", "reference", "changes"];
+    main := ["tutorial", "reference", "changes", "HPC-GAP", "development" ];
```

Comment:
Was this tested? Shouldn't all entries be lower case, to match the output of `SIMPLE_STRING`?

Reply:
yes - for example, in HPC-GAP we have for `??atomic`:

```
[1] Reference: atomic irrationalities
[2] HPC-GAP: The atomic statement.
[3] HPC-GAP: Atomic functions
[4] HPC-GAP: Atomic objects
[5] HPC-GAP: Atomic lists
[6] HPC-GAP: Atomic records and component objects
[7] HPC-GAP: AtomicIncorporateObj
[8] HPC-GAP: AtomicList
[9] HPC-GAP: AtomicList for a count and an object
[10] HPC-GAP: FixedAtomicList
[11] HPC-GAP: FixedAtomicList for a count and an object
[12] HPC-GAP: MakeFixedAtomicList
[13] HPC-GAP: FromAtomicList
[14] HPC-GAP: ATOMIC_ADDITION
[15] HPC-GAP: AtomicRecord
[16] HPC-GAP: AtomicRecord for a record
[17] HPC-GAP: FromAtomicRecord
[18] HPC-GAP: MakeWriteOnceAtomic
[19] HPC-GAP: MakeStrictWriteOnceAtomic
[20] HPC-GAP: MakeReadWriteAtomic

```
and for `??GASMAN`:

```
[1] Reference: GASMAN
[2] Development: GASMAN
[3] Reference: The GASMAN Interface for Weak Pointer Objects
[4] Reference: GasmanStatistics
[5] Reference: GasmanMessageStatus
[6] Reference: SetGasmanMessageStatus
[7] Reference: GasmanLimits
[8] Development: GASMAN Interface
[9] Development: One More Bit of GASMAN interface
```

OTOH, other books are added with commands like `HELP_ADD_BOOK("Tutorial", "GAP 4 Tutorial", "doc/tut");` and then their name is used in lowercase in sortfun, so I will update the spelling for consistency.